src/nemo_safe_synthesizer/llm/metadata.py

```
@@ -882,46 +914,28 @@ class SmolLM2(ModelMetadata):
         **kwargs: Forwarded to [`ModelMetadata`][nemo_safe_synthesizer.llm.metadata.ModelMetadata].
     """
 
+    prompt_template: ClassVar[str] = "user\n {instruction} {schema} \n assistant\n{prefill}"
+    add_bos_token_to_prompt: ClassVar[bool] = False
+    add_eos_token_to_prompt: ClassVar[bool] = False
+    bos_token: ClassVar[str | None] = "<|im_start|>"
```

Ruff (0.15.20)

[error] Possible hardcoded password assigned to: "bos_token"

(S105)